src/objects.c

```diff
@@ -809,10 +809,13 @@ void (*MakeImmutableObjFuncs[LAST_REAL_TNUM+1])( Obj );
 void MakeImmutable(Obj obj)
 {
     if (IS_MUTABLE_OBJ(obj)) {
-        SET_OBJ_FLAG(obj, OBJ_FLAG_IMMUTABLE);
+        if (!IS_INTOBJ(obj) && !IS_FFE(obj)) {
```

Comment:
Neither INTOBJ not FFE can ever be mutable, so I'm not sure what this check is for.